source/dashboard/lcd/lcd.h

```diff
@@ -168,6 +167,10 @@ typedef struct {
 
 } tv_settings_t;
 
+typedef struct {
+  uint16_t brake_bias_adj;  // 0 to 10000 for the page
```

Comment:
If there are other variables similar to this that need to be stored we should add it to this struct to keep everything organized.

Reply:
Will explore

Reply:
Explored: other items on race page for now come directly from can signals, so not necessary to add to struct yet